components/product-details/index.js

```diff
@@ -3,51 +3,55 @@ import personData from '../../mock/person.json';
 import PropTypes from 'prop-types';
 import path from 'path';
 import Image from 'next/image';
+import Link from 'next/link';
+import { Footer } from '../../components/footer';
+import styles from './product-details.module.css';
+
+const renderUses = (productJSON) =>
+  productJSON.usage.map((use, id) => <p key={id}> {use} </p>);
+
+const renderThumbImage = (productJSON) => (
+  <img src={path.join('/assets', productJSON.image)} alt={productJSON.id} />
```

Comment:
I think we don't need node's `path` module here. The same effect should be possible using NextJS's [recommendation](https://nextjs.org/docs/basic-features/static-file-serving)

Reply:
Made the following changes to the entire file